### DISCUSSION TITLE
How to use the remember_me_token with API tokens?

### DISCUSSION THREAD

**Discussion by galaczi** [adonisjs/core#4012] [Help]
How would one use the ``remember_me_token`` with API tokens? Doing the following did not set the ``remember_me_token``. Is it not designed to work with API tokens?

```
    const token = await auth.attempt(credentials.email, credentials.password, {
      expiresIn: '1 day'
    }, true)
```

**Comment by dimitrisnl** [comment]
Isn't the "remember me token", the API token in this case?
